pcdsdevices/utils.py

```diff
@@ -154,15 +155,15 @@ def convert_unit(value, unit, new_unit):
     new_value : float
         The starting value, but converted to the new unit.
     """
+    unit = getattr(units, unit)
+    new_unit = getattr(units, new_unit)
 
-    global ureg
-    if ureg is None:
-        import pint
+    if unit == new_unit:
```

Comment:
I presume this is to catch cases like `convert_unit(1, 's', 'seconds')`?

I also like how this helps us avoid globals.

Reply:
Yes, and further this is because the results of the following conversion don't have a coefficient if you convert the same unit to itself, which was breaking the test suite in an earlier version.